src/Components/DrillPageMinimal.js

```diff
@@ -42,27 +43,35 @@ export const DrillPageMinimal = (props) => {
     const headerTitle = () => (
       <View style={styles.headerTitle}>
         <Text numberOfLines={1} style={styles.headerTitleText}>
-          {I18n.t('drillPageMinimal.headerTitle', { trainingTitle: training.title })}
+          {I18n.t('drillPageMinimal.headerTitle', { trainingTitle: drill.title })}
         </Text>
-        <ProgressBar total={training.drills.length} current={currentDrillIndex + 1} onDotPress={onProgressDotPress} />
+        <View >
+          <ProgressBar total={training.drills.length} current={currentDrillIndex + 1} onDotPress={onProgressDotPress} />
+        </View>
       </View>
     );
+
+    const headerRight = () => {
```

Comment:
I think we decided (you wrote this in the issue #238 ) to remove this button and put the description at the bottom of the screen ?

Reply:
I havn't succeed to to it. The other problem is the scrolldown on ios doesn't work well . We should change it